test/uk/gov/hmrc/cgtpropertydisposals/service/returns/CgtCalculationServiceImplSpec.scala

```diff
@@ -215,8 +217,8 @@ class CgtCalculationServiceImplSpec extends WordSpec with Matchers with ScalaChe
               reliefDetails      = reliefDetails
             )
 
-            result.initialGainOrLoss.value      shouldBe 100L
-            result.gainOrLossAfterReliefs.value shouldBe 0L
+            result.initialGainOrLoss.amount.value shouldBe 100L
```

Comment:
it would be better to assert the source in the `AmountInPenceWithSource` as well - then we should make sure that there is at least one test testing that we get a `Calculated` and at least one for `UserSupplied`

Reply:
ok I'm writing a new test for UserSupplied vals